3_acm/install_acm.sh

```diff
@@ -51,6 +51,9 @@ for CONTEXT in ${CLUSTERS}; do
     cat config-management.yaml.tpl | envsubst > config-management-${CONTEXT}.yaml
     kubectl apply -f config-management-${CONTEXT}.yaml
     rm config-management-${CONTEXT}.yaml
+    cat root-sync.yaml.tpl | envsubst > root-sync-${CONTEXT}.yaml
+    kubectl apply -f root-sync-${CONTEXT}.yaml
```

Comment:
Since this filename is used multiple times, you can put it into an variable.

Reply:
I am talking about `root-sync-${CONTEXT}.yaml`

Reply:
Done